### PR TITLE
initial integration with api

### LINKED ISSUE #23: Write function to add new event to database
Update event_command_end function in event_command.py

### LINKED ISSUE #24: Write function to add new task to database
Update task_schedule_no_update function in task_command.py
Update task_command_end function in task_command.py

### PR DESCRIPTION
**Pull request by s7u4rt99**
Fixes https://github.com/tr3002-ay2324s1/nova/issues/23
Fixes https://github.com/tr3002-ay2324s1/nova/issues/24

### CODE REVIEW

File: commands/task_command.py [hunk 1]
```diff
@@ -180,7 +191,14 @@ async def task_schedule_updated(update: Update, context: ContextTypes.DEFAULT_TY
...
+    data = {
+        "userId": 123,
+        "name": "",
+        "description": "",
+    }
+
+    url_post = f"{os.getenv('REQUEST_URL')}/tasks"
+    requests.post(url_post, json=data)
 
     # TODO: create new event on gcal
```
Comment (s7u4rt99) [inline commands/task_command.py]: @juliussneezer04 can you add this?

Comment (juliussneezer04) [reply commands/task_command.py]: yep on me

File: commands/task_command.py [hunk 2]
```diff
@@ -124,7 +128,14 @@ async def task_schedule_no_update(update: Update, context: ContextTypes.DEFAULT_
         await send_on_error_message(context)
         return
 
-    # TODO: add new task to database
+    data = {
```
Comment (s7u4rt99) [inline commands/task_command.py:154]: @joshuan98 how do i get the fields for the data? do we even have description and user id?

File: commands/event_command.py [hunk 3]
```diff
@@ -121,7 +125,14 @@ async def event_command_end(update: Update, context: ContextTypes.DEFAULT_TYPE):
         await send_on_error_message(context)
         return
 
-    # TODO: add new event to database
+    data = {
```
Comment (s7u4rt99) [inline commands/event_command.py:128]: same

File: commands/task_command.py (hunk 2 above)
Comment (juliussneezer04) [reply commands/task_command.py:154]: i can handle this, will update when I add the g cal functionality cause then we'll know what to add in

**Review by juliussneezer04 — APPROVED** [review]
looks good, just need to add in my g cal work to make these work

**Pull request merged**